consensus/src/dag/dag_network.rs

```diff
@@ -42,6 +43,10 @@ pub trait TDAGNetworkSender: Send + Sync + RBNetworkSender<DAGMessage> {
         retry_interval: Duration,
         rpc_timeout: Duration,
     ) -> RpcWithFallback;
+
+    async fn send_epoch_change(&self, proof: EpochChangeProof);
+
+    async fn send_commit_proof(&self, ledger_info: LedgerInfoWithSignatures);
```

Comment:
I don't want to couple this two with network sender, this is more like self notifier. maybe another adapter trait?